packages/hash/frontend/src/blocks/page/PageTitle/PageTitle.tsx

```diff
@@ -107,7 +110,7 @@ export const PageTitle: FunctionComponent<PageTitleProps> = ({
     <StyledTextarea
       ref={pageTitleRef}
       placeholder={PAGE_TITLE_PLACEHOLDER}
-      disabled={updateEntityLoading}
+      disabled={updateEntityLoading || readonlyMode}
```

Comment:
```suggestion
      readOnly={readOnlyMode}
      disabled={updateEntityLoading}
```

Reply:
Hmm, a `disabled` text area means "non editable" text area right? So don't want users to edit the text area when loading, and when it's on readonly mode. So I feel like current implementation makes sense and simple enough. Let me know if you have another concern regarding this.